packages/browser/package.json

```diff
@@ -81,14 +78,17 @@
     "@types/node-jose": "^1.1.5",
     "@types/uuid": "^8.3.0",
     "build-module": "^1.0.7",
+    "crypto-browserify": "^3.12.0",
```

Comment:
I 'guess' this might be Ok, but I thought `browserify` was long dead nowadays. It seems this repo (https://github.com/crypto-browserify/crypto-browserify) hasn't been updated in 3 years too, and this a dependency, not a Dev dependency. Anyway, just seems a tad curious (when I thought we were using native browser crypto, yet this package states "**_...reimplement node's crypto module, in pure javascript so that it can run in the browser_**")...?

Reply:
That's the problem: right now, we aren't using native browser crypto, `node-jose` and `jsonwebtoken` depend on these node builtins. Previous versions of Webpack used to polyfill these dependencies, but they no longer do starting on version 5. `crypto-browserify` is their recommanded replacement.